Dockerfile

```diff
@@ -6,6 +6,9 @@ ENV AIRFLOW_UID=501
 COPY requirements.txt ./requirements.txt
 COPY requirements-test.txt ./requirements-test.txt
 COPY dags ./dags
-
+RUN pip install wheel
+RUN python setup.py bdist_wheel 
+RUN pip install --no-cache-dir --upgrade setuptools && \
+    pip install --no-cache-dir --user -r requirements.txt -r requirements-test.txt
 USER airflow
-RUN pip install --no-cache-dir --user -r requirements.txt -r requirements-test.txt
+
```

Comment:
Adding multiple `RUN` adds layer to docker hence it makes the image build slower. You can combine all the commands into one. What I don't understand is why we have to install wheel, run  python setup.py bdist_wheel  and upgrade setuptools? This question also applies to the workflows file.

Reply:
it was an overdone thing, in order to solve a problem. We don't need to install them, we need just upgrade